tests/samplers_tests/test_nsgaiii.py

```diff
@@ -205,6 +209,36 @@ def test_constraints_func_experimental_warning() -> None:
         NSGAIIISampler(constraints_func=lambda _: [0])
 
 
+def test_child_generation_strategy_experimental_warning() -> None:
+    with pytest.warns(optuna.exceptions.ExperimentalWarning):
+        NSGAIIISampler(child_generation_strategy=lambda study, search_space, parent_population: {})
+
+
+def test_sample_relative() -> None:
```

Comment:
Ditto. 

I think it might be removed since it is overlapped with that defined in `test_nsgaii.py`.

Reply:
I removed the test.